# Week of 2023-04-17 to 2023-04-24 (39)

### GHFirst (10)

- [Revert "Reland python ops (#99170)"](https://github.com/pytorch/pytorch/commit/1c042a21370be75d9cd9581a520c392bae466171)
  - Break internal build ([comment](https://github.com/pytorch/pytorch/pull/99170#issuecomment-1512922761))
- [Revert "[c10d] Faster coalescing (#98793)"](https://github.com/pytorch/pytorch/commit/9861ec9785b53e71c9affd7d268ef7073eb1c446)
  - Break internal build ([comment](https://github.com/pytorch/pytorch/pull/98793#issuecomment-1517526663))
- [Revert "[quant][pt2e][refactor] Cleanup the logic for deciding whether to insert observer/fq or not (#99220)"](https://github.com/pytorch/pytorch/commit/75e754800f42631425a483cdcfa0b34e8e14d659)
  - Diff reverted internally ([comment](https://github.com/pytorch/pytorch/pull/99220#issuecomment-1518166594))
- [Revert "Use correct standard when compiling NVCC on Windows (#99492)"](https://github.com/pytorch/pytorch/commit/cfacb5eaaa1d5eda92806cd5f23d3f69817f19a1)
  - Diff reverted internally ([comment](https://github.com/pytorch/pytorch/pull/99492#issuecomment-1515360979))
- [Revert "fix onednn ConvTranspose2d channels last issue when ic=1 (#99539)"](https://github.com/pytorch/pytorch/commit/da57d597e11c52a4a33c8d268561a36c57725149)
  - Break internal build ([comment](https://github.com/pytorch/pytorch/pull/99539#issuecomment-1517484303))
- [Revert "Switch calling convention back to real tensors (#99320)"](https://github.com/pytorch/pytorch/commit/ea50d4f1460396dc52d23a1c80bcc56405688c0f)
  - Break internal build ([comment](https://github.com/pytorch/pytorch/pull/99320#issuecomment-1514437071))
- [Revert "Python binding to set/get CUDA rng state offset (#98965)"](https://github.com/pytorch/pytorch/commit/bb2cd4a107d1913335d9cd4dfcfff3906f1382dd)
  - Break internal build ([comment](https://github.com/pytorch/pytorch/pull/98965#issuecomment-1514561840))
- [Revert "[quant][fix] Compare resnet with quantizer api with the prepare_fx and decomposed convert flow (#98905)"](https://github.com/pytorch/pytorch/commit/20a1788136166ace127c12d1e8f38779be1a5e34)
  - Conflicts with D44918496 landed internally, blocks diff train import ([comment](https://github.com/pytorch/pytorch/pull/98905#issuecomment-1510533477))
- [Revert "Disable dynamo tracing torchrec.distributed (#97824)"](https://github.com/pytorch/pytorch/commit/ab08284225406238d2077ebcce165c788aafe837)
  - back out diff that doubles memory consumption for multitask FAIM flows. See D44978317 ([comment](https://github.com/pytorch/pytorch/pull/97824#issuecomment-1512048303))
- [Revert "Simple Custom Operator API, V0 (#98440)"](https://github.com/pytorch/pytorch/commit/f497031df99f44e410605de3220d108517713a40)
  - Break internal build ([comment](https://github.com/pytorch/pytorch/pull/98440#issuecomment-1513090063))

### Ignored Signal (4)

- [Revert "Codegen fixed size for static sympy values (#99362)"](https://github.com/pytorch/pytorch/commit/bb017d7671869ca3a6e8268ede192a3676aaaf2b)
  - CI Fail ([comment](https://github.com/pytorch/pytorch/pull/99362#issuecomment-1513973050))
- [Revert "Fix bug in check required output size in _as_strided_scatter_meta (#98483)"](https://github.com/pytorch/pytorch/commit/bce21ee06ab5657259f5e1f23499a1f25cd64afd)
  - Broke inductor, see https://hud.pytorch.org/hud/pytorch/pytorch/main/1?per_page=50&name_filter=inductor%2C%201%2C%201 ([comment](https://github.com/pytorch/pytorch/pull/98483#issuecomment-1513654507))
- [Revert "[cuda rng] Making offset calculation independent of device properties (#98988)"](https://github.com/pytorch/pytorch/commit/5cb788a9a5c00a20116e8f6141d2a8146eb0f887)
  - Diagnosing if sebotnet has flakiness ([comment](https://github.com/pytorch/pytorch/pull/98988#issuecomment-1515099552))
- [Revert "Codegen fixed size for static sympy values (#99362)"](https://github.com/pytorch/pytorch/commit/19501b254f753218520a51c67245164d56d6d2e2)
  - Reverting, as it breaks lint ([comment](https://github.com/pytorch/pytorch/pull/99362#issuecomment-1513650318))

### Landrace (2)

- [Revert "Mark doctr_det_predictor as broken on master (#99370)"](https://github.com/pytorch/pytorch/commit/ce7c4ba11de03953c037dfe6e3f587d2c4807ff9)
  - malfet already directly fixed it ([comment](https://github.com/pytorch/pytorch/pull/99370#issuecomment-1513128507))
- [Revert "Allow in graph einops operators (#99478)"](https://github.com/pytorch/pytorch/commit/96a262d666766342ea7a3e26d43a734b4ee2d5c2)
  - dynamo/test_after_aot.py::TestAfterAot::test_save_graph_repro - AssertionError, see https://github.com/pytorch/pytorch/actions/runs/4750274195/jobs/8438535867 ([comment](https://github.com/pytorch/pytorch/pull/99478#issuecomment-1515788708))

### Not through pytorchbot (3)

- [Revert "Build Windows binaries with Visual Studio 2022 Build Tools (#90855) (#99591)](https://github.com/pytorch/pytorch/commit/7c3fa5c70dbbdc45bc51b29773753da966583483)
- [Revert "Revert "Expandable blocks in allocator (#96995)"" (#99275)](https://github.com/pytorch/pytorch/commit/7ff1f3f3f696de97a678e305b563e8ee2a317117)
- [Revert D44897935: Multisect successfully blamed D44897935 for test or build failures (#99353)](https://github.com/pytorch/pytorch/commit/ccc5d1daec46da82ce17fcb8e9dcc871e9fef9a2)

### No Signal (5)

- [Revert "[inductor] coordinate descent tuning upon max-autotune (#97203)"](https://github.com/pytorch/pytorch/commit/4aedb8e11664cc660b89b2dcecf2875ea5fa3ced)
  - Sorry for reverting your PR, but it breaks MacOS test in trunk ([comment](https://github.com/pytorch/pytorch/pull/97203#issuecomment-1514047636))
- [Revert "Simplify _use_grad_for_differentiable (#98706)"](https://github.com/pytorch/pytorch/commit/4637c5ae5bb5eeb35e39f14c63f8b104b495fb11)
  - Sorry for reverting your PR but a bunch of inductor tests are failing after this commit, so reverting the PR just to be sure ([comment](https://github.com/pytorch/pytorch/pull/98706#issuecomment-1518444021))
- [Revert "[pt2] add `SymInt` support for `column_stack` (#99276)"](https://github.com/pytorch/pytorch/commit/08dd4ad0b91593a87a423d313cff4e93a78234f3)
  - reverting this one too for safety ([comment](https://github.com/pytorch/pytorch/pull/99276#issuecomment-1511972021))
- [Revert "[vmap] Fix searchsorted batch rule for self_logical_rank == 0 (#99526)"](https://github.com/pytorch/pytorch/commit/35ad5122d2e71737aaaf982e1f3a6c8f681fefda)
  - Regressed behavior ([comment](https://github.com/pytorch/pytorch/pull/99526#issuecomment-1516313920))
- [Revert "[pt2] add meta function for `linalg.cross` (#99279)"](https://github.com/pytorch/pytorch/commit/f957334c2bef4689642796f1d7430c6946bc0083)
  - Apparently this is breaking inductor on master? So weird ([comment](https://github.com/pytorch/pytorch/pull/99279#issuecomment-1511967868))

### Got @pytorchbot revert command, but no corresponding commit (9)

- cannot find commit corresponding to @pytorchbot revert comment
  - Reverting the previous PR ([comment](https://github.com/pytorch/pytorch/pull/99581#issuecomment-1515791845))
- cannot find commit corresponding to @pytorchbot revert comment
  - No longer needed ([comment](https://github.com/pytorch/pytorch/pull/99579#issuecomment-1515808478))
- cannot find commit corresponding to @pytorchbot revert comment
  - Lint caught several issues ([comment](https://github.com/pytorch/pytorch/pull/99220#issuecomment-1518149854))
- cannot find commit corresponding to @pytorchbot revert comment
  - dynamo/test_after_aot.py::TestAfterAot::test_save_graph_repro - AssertionError, see https://github.com/pytorch/pytorch/actions/runs/4750274195/jobs/8438535867 ([comment](https://github.com/pytorch/pytorch/pull/99478#issuecomment-1515790998))
- cannot find commit corresponding to @pytorchbot revert comment
  - Break internal build ([comment](https://github.com/pytorch/pytorch/pull/98590#issuecomment-1517555266))
- cannot find commit corresponding to @pytorchbot revert comment
  - CI Fail ([comment](https://github.com/pytorch/pytorch/pull/99362#issuecomment-1513972636))
- cannot find commit corresponding to @pytorchbot revert comment
  - No longer needed ([comment](https://github.com/pytorch/pytorch/pull/99579#issuecomment-1515820722))
- cannot find commit corresponding to @pytorchbot revert comment
  - Break internal build ([comment](https://github.com/pytorch/pytorch/pull/99320#issuecomment-1516044479))
- cannot find commit corresponding to @pytorchbot revert comment
  - back out diff that doubles memory consumption for multitask FAIM flows ([comment](https://github.com/pytorch/pytorch/pull/97824#issuecomment-1511814626))

### Weird (6)

- [Revert "Skip sccache initialization on MacOS (#99121)"](https://github.com/pytorch/pytorch/commit/6212a85af83a018b3d8645edfebfb19f1c984155)
  - The cache still timeout and there is no way to increase the timeout value to be more than 10s looking at sccache code https://github.com/mozilla/sccache/blob/6bbef54b886254ce26968a0cce1e88f4a774c1bf/src/commands.rs#L48.  This needs reworks ([comment](https://github.com/pytorch/pytorch/pull/99121#issuecomment-1511792300))
- [Revert "Fix trailing spaces lint (#99581)"](https://github.com/pytorch/pytorch/commit/21e88a543b98b7df367b4b3a343a8b2d582be455)
  - Reverting the previous PR ([comment](https://github.com/pytorch/pytorch/pull/99581#issuecomment-1515817141))
- [Revert "Skip sccache initialization on MacOS (#99121)"](https://github.com/pytorch/pytorch/commit/08ffe34621ded20cdb0825a890b334354a06e03c)
  - Revert to reland this as this miss one change in mac build workflow. My mistake from rebasing from master into main ([comment](https://github.com/pytorch/pytorch/pull/99121#issuecomment-1512264581))
- [Revert "Temporarily move ROCm to unstable (#99579)"](https://github.com/pytorch/pytorch/commit/4d8906885e672d088849303cc8e821cff81ba6d9)
  - No longer needed ([comment](https://github.com/pytorch/pytorch/pull/99579#issuecomment-1515792486))
- [Revert "[inductor] adjust sliceView limits (#99447)"](https://github.com/pytorch/pytorch/commit/96cad5cf95cec565e2d4af74d28c6ba838132c7d)
  - breaks inductor ([comment](https://github.com/pytorch/pytorch/pull/99447#issuecomment-1514013091))
- [Revert "Disable XProtect on MacOS runner (#99506)"](https://github.com/pytorch/pytorch/commit/fc63d710fee323eee8b135fd193ee37e9f06ed55)
  - Found a clue on the uploaded archive, reverting this so I can update the PR with the correct mitigation ([comment](https://github.com/pytorch/pytorch/pull/99506#issuecomment-1518252418))
